deeplearning4j-core/src/main/java/org/deeplearning4j/nn/conf/preprocessor/UnitVarianceProcessor.java

```diff
@@ -28,8 +32,11 @@
  *
  * @author Adma Gibson
  */
+@Data
 public class UnitVarianceProcessor extends BaseInputPreProcessor {
 
+    @Getter(AccessLevel.NONE)
+    @Setter(AccessLevel.NONE)
```

Comment:
These annotations are needed to hide the columnStds field.   It is not really a part of the configuration, rather it looks to be internal state kept during execution.